mmv1/templates/terraform/constants/spanner_instance_config.go.erb

```diff
@@ -0,0 +1,57 @@
+<%# The license inside this block applies to this file
+	# Copyright 2021 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func replicasHash(v interface{}) int {
+	var buf bytes.Buffer
+	m := v.(map[string]interface{})
+	buf.WriteString(fmt.Sprintf("%s-", strings.ToLower(m["location"].(string)))) // ToLower just in case
+	buf.WriteString(fmt.Sprintf("%s-", strings.ToLower(m["type"].(string))))
+	var isLeader interface{}
+	if m["defaultLeaderLocation"] != nil {
+		isLeader = m["defaultLeaderLocation"]
+	} else {
+		isLeader = false
+	}
+	buf.WriteString(fmt.Sprintf("%v-", isLeader.(bool)))
+	return tpgresource.Hashcode(buf.String())
+}
+
+func getBaseInstanceConfigReplicas(d *schema.ResourceData, config *transport_tpg.Config, baseConfigProp interface{}, billingProject, userAgent string) ([]interface{}, error) {
```

Comment:
```suggestion
func getBaseInstanceConfigReplicas(d *tpgresource.TerraformResourceData, config *transport_tpg.Config, baseConfigProp interface{}, billingProject, userAgent string) ([]interface{}, error) {
```

Should fix the unit test error: https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/9372875007/job/25805327976#step:7:152